packages/gafl-webapp-service/src/pages/summary/licence-summary/route.js

```diff
@@ -50,14 +50,22 @@ const getData = async request => {
 
   await findPermit(permission, request)
 
-  // Is this date format ok?
-  const startTimeString = moment(permission.licenceStartDate, 'YYYY-MM-DD')
-    .add(permission.licenceStartTime, 'hours')
-    .format('LLLL')
+  const startDateString = moment(permission.licenceStartDate, 'YYYY-MM-DD').format('dddd, MMMM Do, YYYY')
```

Comment:
Shouldn't be using Midnight and Midday - I appreciate this is a little up in the air at the moment but the improvements made to the current service to use 12:01 etc were well received in user feedback.
We should use this as a starting point until there is evidence for something better.

Reply:
Its wrong though isn't it. The licence starts at 12:00. That is when you have asked for it to start.

Reply:
You have asked for it to start my selecting the 'Midnight' radio button. Lets pass it to Jon

Reply:
I am uncomfortable doing this. Regardless of any 'user research' or previously established rule this is incorrect a priori. On a previous page you have given the user the option to select midday as a licence start time and this is what should be played back. IMHO it looks ridiculous to then translate that into 12:01pm. If it is 12:01 pm then the radio button should be label that way. Let us tell the user accurately when his licence starts and ends please.

Reply:
I agree with you that it should be consistent throughout the journey and any communications but I don't think we can revert back to something that was less favourable with our users especially when we have metrics to support the change to use 12:01 etc.

Reply:
I've created https://github.com/DEFRA/rod-licensing/issues/137 to track this.